save-backend/src/main/kotlin/com/saveourtool/save/backend/SaveApplication.kt

```diff
@@ -10,7 +10,7 @@ import org.springframework.context.annotation.Import
 /**
  * An entrypoint for spring for save-backend
  */
-@SpringBootApplication(scanBasePackages = ["com.saveourtool.save.backend", "com.saveourtool.save.repository"])
+@SpringBootApplication(scanBasePackages = ["com.saveourtool.save.backend", "com.saveourtool.save"])
```

Comment:
looks like the problem that `com.saveourtool.save` is subset of `com.saveourtool.save.backend`.
It does work if packages specify explicitly:
```kotlin
@SpringBootApplication(scanBasePackages = [
    "com.saveourtool.save.backend",
    "com.saveourtool.save.service",
    "com.saveourtool.save.security",
    "com.saveourtool.save.storage",
    "com.saveourtool.save.utils",
])
```